.gitattributes

```diff
@@ -0,0 +1,2 @@
+sbt
```

Comment:
Good catch! However, to make it work it should include `linguist-vendored` as per official [docs](https://github.com/github/linguist#vendored-code).